controllers/profiles/dev/object_creators_dev.go

```diff
@@ -95,7 +96,7 @@ func deploymentMutateVisitor(workflow *operatorapi.SonataFlow, plf *operatorapi.
 	}
 }
 
-func ensureWorkflowDefConfigMapMutator(workflow *operatorapi.SonataFlow) common.MutateVisitor {
+func ensureWorkflowDefConfigMapMutator(workflow *operatorapi.SonataFlow, support *common.StateSupport) common.MutateVisitor {
```

Comment:
```suggestion
func ensureWorkflowDefConfigMapMutator(workflow *operatorapi.SonataFlow) common.MutateVisitor {
```

I think we don't need this attribute anymore.

Reply:
Correct. Removed.